.github/workflows/update-helm-repo.yaml

```diff
@@ -94,8 +94,7 @@ jobs:
     needs: [setup]
     runs-on: ubuntu-latest
     permissions:
-      contents: write # to push chart release, create release, and push tags to github
```

Comment:
we still create a GH release. Don't we still need `contents`

https://github.com/grafana/helm-charts/blob/226cf91bb0222cc2a2cc66449a24395e7b041127/.github/workflows/update-helm-repo.yaml#L205-L219

Reply:
As I understand it, the `permissions` only apply to `GITHUB_TOKEN`. But this workflow publishes the content via the app install token, which has the scope assigned when the app is installed.

Not that the workflow never had `permissions`. They were added only recently in the https://github.com/grafana/helm-charts/pull/3366 (I don't think this was necessary)